mods/sgssb/moves.js

```diff
@@ -615,7 +615,7 @@ exports.BattleMovedex = {
 		id: "aquasubscribe",
 		name: "Aqua Subscribe",
 		priority: 1,
-		secondary: {
+		self: {
```

Comment:
Regardless, I need to test this before I implement, a formats crash requires a restart most of the time.